### DISCUSSION TITLE
error starting serene mvc project

### DISCUSSION THREAD

**Discussion by zprofreelancer** [serenity-is/Serenity#5376] [Help] [answered]
Hello,
I'm using visual studio 2017 and have mssqllocaldb installed. But when i run the old serene mvc project, it issues "A network-related or instance-specific error occurred"
i checked the connection string in web.config file and it uses datasource localdb\mssqllocaldb. This should be correct.
So, what could be the problem?. please help

thanks.

**Comment by zprofreelancer** [comment]
after starting visual studio as administrator and creating the project, now after running the project it issues "sql exception. connection timeout expired"

please help

**Comment by minhhungit** [comment, marked as answer]
>A network-related or instance-specific error occurred

normally due to wrong connectionstring, or sql server is stopped, so try to use Server Explorer or another tool to connect to your database


>after starting visual studio as administrator and creating the project, now after running the project it issues "sql exception. connection ?timeout expired"

Can you show more detail about your exception, show your code, take a screenshot ...

**Reply by zprofreelancer** [reply]
problem solved. Thanks....the problem was because of unstable sql server service.

**Reply by minhhungit** [reply]
@zprofreelancer  if it works please mark this discussion as answered, thank you
